lib/goals/watch-sync.js

```diff
@@ -0,0 +1,108 @@
+/*
+ *
+ *  Copyright 2016-2017 Red Hat, Inc, and individual contributors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+'use strict';
+
+// This is the goal for connecting to a running pod and doing an "oc rsync" when files chnage locally
+const logger = require('../common-log')();
+const { spawn } = require('child_process');
+
+// TODO: move this to the helpers?
```

Comment:
Should you just move this as a part of this PR?

Reply:
Yea, i think that makes sense.  I think i was going to make it a separate commit and not squash it, but then i think i forgot to add it :)